src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,21 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+#### Asking complex questions without using hint text
+
+Do not use hint text if you need to give a lengthy explanation with lists and paragraphs. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+You should also avoid using links in hint text, as screen readers might not read out them out.
+
+If you're asking a question that needs a detailed explanation, use:
+* a `h1` heading that's a statement (for example, 'Add undergraduate degree') rather than a question
+* whatever mix of text, paragraphs, lists and examples best explains your question to users
+* a label, above the form input, that asks users a specific question – for example, 'What type of degree do you want to add?'
```

Comment:
Update to match the example?

```suggestion
* a `h1` heading that's a statement (for example, 'Interview needs') rather than a question
* whatever mix of text, paragraphs, lists and examples best explains your question to users
* a label, above the form input, that asks users a specific question – for example, 'Do you have any interview needs?'
```